composer.lock

```diff
@@ -8,16 +8,16 @@
     "packages": [
         {
             "name": "beberlei/DoctrineExtensions",
-            "version": "v1.0.19",
```

Comment:
Any particular reason to update dependencies?

Reply:
Yep, I wanted to get the modules compatible with PHP 7.2. Upgraded with the merge of https://github.com/PrestaShop/ps_facetedsearch/pull/30

Reply:
In the future you can update PrestaShop's own dependencies only by running
```
composer update "prestashop/*"
```
This way we won't risk regressions from updated dependencies which are unrelated